scripts/CMSRunAnalysis.py

```diff
@@ -933,11 +911,11 @@ def StripReport(report):
         setupLogging('.')
 
         # Also add stdout to the logging
```

Comment:
Do you remember why these logging lines were added?

Reply:
I suspect that it was so that earlier version of scram() would send to stdout and hence to the file that we use, instead of logging only to wmagent.log. With new scram() the command output is received in a memory buffer instead and we later fetch it via scram.stdout()

Reply:
I am adding a not in the comment